tests/test_ir/test_deserializer.py

```diff
@@ -225,7 +225,7 @@ def test_deserializer_with_random_logs(self) -> None:
             )
 
 
-class TestCaseDeserializerDecompress(TestCaseDeserializerBase):
+class TestCaseFourByteDeserializerDecompress(TestCaseFourByteDeserializerBase):
```

Comment:
Nit (and you can ignore this if you don't agree with me)
The names look terribly long for me in this file. It should be clear that if the base class is four bytes, then inherited class will also be testing four bytes.
Do we have a plan to support 8bytes in the future? If not, maybe we can remove "FourByte" from the inhertied class. If we have such a plan, maybe it would worth splitting the test file into 2, one for 4 bytes and one for 8 bytes?

Reply:
no, we don't have plan to support 8byte...
We will rewrite most of these functions for v2 format. Once that format is formally released, all v1 related code will be removed from the code base.
I will pay more attentions to test case namings for v2 unit tests :)